R/export_quiz.R

```diff
@@ -0,0 +1,35 @@
+#' Export quiz as TeX
+#'
+#' @param quiz a data.frame created by import_quiz
+#' @param output_file the location to save the TeX document
+#' @param class_name name of class to be used on quiz
+#' @param quiz_name name of quiz to be used on quiz
+#' @param n_ver number of versions to create when TeX is compiled
+#' @return outputs a TeX file to output_file location
+#' @export
+export_quiz <- function(quiz, output_file, class_name, quiz_name, n_ver){
```

Comment:
Shouldn't `class_name` and `quiz_name` default to `""` and `n_ver` default to 1?